src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,25 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
```

Comment:
The method 'register' should have a more descriptive name according to the Java naming conventions, such as 'registerUser' to clarify the action being performed.